### DISCUSSION TITLE
请问openclash如何配置才可以正常使用网游加速器？

### DISCUSSION THREAD

**Discussion by EliteOtaku** [vernesong/OpenClash#3577] [General]
我的网络环境如下：
移动宽带，光猫设置为桥接模式，软路由为
![image](https://github.com/vernesong/OpenClash/assets/29235152/9096b788-14e2-4efc-8bf2-65ec092b9b82)
软路由负责拨号，安装了openclash
![image](https://github.com/vernesong/OpenClash/assets/29235152/036890aa-db44-4471-bdba-5862187fe4bd)
![image](https://github.com/vernesong/OpenClash/assets/29235152/a2fc78d1-9327-4a86-a30d-89b81b2f7c5c)
![image](https://github.com/vernesong/OpenClash/assets/29235152/bf3219e8-014b-4e9d-a8c4-f985bd34437e)
![image](https://github.com/vernesong/OpenClash/assets/29235152/129d6925-6d9b-4b6e-97bc-a66cba2a7b6b)
![image](https://github.com/vernesong/OpenClash/assets/29235152/44b7f9b1-4618-4bac-be5a-974b546ba9b0)
![image](https://github.com/vernesong/OpenClash/assets/29235152/f85ec9c3-9c66-4415-bb84-21ddc1806c59)
![image](https://github.com/vernesong/OpenClash/assets/29235152/a48b9ce3-bbac-4bdf-968c-71be700769db)
平时使用的时候正常，但是用网易UU加速器的时候，比如想加速韩服，那么加速节点有比较大概率会自动选择到香港/台湾的节点，推测是科学上网的服务器节点，然后需要手动选择韩国节点才可以，但是游戏加速功能是正常的。
![image](https://github.com/vernesong/OpenClash/assets/29235152/03c8037a-2fe7-475a-bc10-b1641895d56a)
最近比较烦的是，加速战地2042的时候，会不定时出现输入指令无响应的情况，就是大概每隔2-5分钟会突然发现鼠标键盘的输入完全停止1-2秒，然后再恢复正常，有时候刚好在对枪时就死不瞑目了……T_T
我试过Meta with fakeip/redir-host，分别的2x2，共4个组合，都会出现这样的问题，但是我不知道在哪里可以查看相关日志，内核版本也全部更新过。如果关闭openclash，直接加速，那么问题就不会出现，但我的工作刚好需要长时间科学上网，所以很是头疼，希望高手能帮忙解答一下，或者给一点解决思路，非常感谢。

**Comment by EliteOtaku** [comment]
[OpenClash 调试日志.txt](https://github.com/vernesong/OpenClash/files/13160800/OpenClash.txt)

**Comment by AFeixue** [comment]
我也是这样  大哥你解决了吗

**Comment by ghost** [comment]
uu里有一个`仅限长宽`的选项 具体忘了叫啥了 勾上试试

**Reply by UntR** [reply]
解决了，谢谢大佬

**Comment by vernesong** [comment]
uu加速正常不会受影响，你全局直连试试，没问题就是你的大陆ip库有问题

**Reply by AFeixue** [reply]
没用UU玩 我奇游VK加速器都这样 晚点我去买个UU试试看
